app/components/VideoSet.js

```diff
@@ -16,30 +17,96 @@ interface Props {
   thirdVideoType: string;
   fourthVideo: string;
   fourthVideoType: string;
+  isRunning: boolean;
+  questionAlreadyShown: boolean;
+  answer1: string;
 }
 
+const controlPauseTime = 4;
+
 export default class VideoSet extends Component<Props> {
   propTypes: {
     location: React.PropTypes.object
   };
 
   props: Props;
 
+  constructor(props) {
+    super(props);
+    // This binding is necessary to make `this` work in the callback
+    this.playVideo = this.playVideo.bind(this);
+    this.pauseVideo = this.pauseVideo.bind(this);
+    this.handleQuestion1 = this.handleQuestion1.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+
+    this.state = {
+      isRunning: 'false',
+      questionAlreadyShown: 'false',
+      answer1: 'option5'
+    };
+  }
+
   state = {
-    openModal: false
+    modalIsOpen: false,
+    isRunning: false,
+    questionAlreadyShown: false
   };
 
-  close = () => this.setState({ openModal: false });
+  closeModal = () => this.setState({ modalIsOpen: false });
 
-  open = () => this.setState({ openModal: true });
+  openModal = () =>
+    this.setState({
+      modalIsOpen: true
+    });
 
-  render() {
-    const { openModal } = this.state;
+  playVideo = () => {
+    const videoRef = this.getVideoRef;
+    videoRef.play();
+    this.setState({ isRunning: true });
+  };
 
-    const { location } = this.props;
+  pauseVideo = () => {
+    const videoRef = this.getVideoRef;
+    videoRef.pause();
+    this.setState({ isRunning: false });
+  };
 
-    const { state } = location;
+  onTimeUpdate = () => {
```

Comment:
This looks good. We'll probably spend a lot of time tweaking and adding stuff on to this function as we develop the control and experimental paradigms